app/Controllers/authController.php

```diff
@@ -151,7 +155,10 @@ public function formLoginAction() {
 				                  ' user=' . $username .
 				                  ', nonce=' . $nonce .
 				                  ', c=' . $challenge);
-				Minz_Error::error(403, array(_t('feedback.auth.login.invalid')), false);
+				Minz_Request::bad(
```

Comment:
The new code returns 200, and not 403 anymore, right? It could be good to reintroduce the 403 at some point.
Indeed, error codes are used by other systems (such as fail2ban) e.g. to detect attacts

Reply:
To be exact it returns a 302. I think there's work to do on how we handle responses, and errors in particular but I didn't want to make this PR too complicated. Here I wanted to keep the users on the same URL (`?c=auth&a=login`) for consistency and because there's a small condition that would be skipped otherwise (nothing dramatic).

It wouldn't be too complicated to rework this part, though I'm not sure how important it is.

Reply:
https://github.com/FreshRSS/FreshRSS/issues/2901

Reply:
https://github.com/FreshRSS/FreshRSS/pull/2903